src/maps/components/AddressPicker.vue

```diff
@@ -126,10 +128,18 @@ export default {
       default: null,
       type: String,
     },
+    outlined: {
+      default: false,
+      type: Boolean,
+    },
     defaultMapCenter: {
       default: null,
       type: Object,
     },
+    tmpNewBorderStyle: {
+      default: false,
+      type: Boolean,
+    },
```

Comment:
I first felt _ugughuhruh_ about this prop name, with the _tmp_ bit in it, but I see the idea, it forces us to acknowledge it is temporary, and maybe a bit more motivation to fix it soon! So I approve :)

Reply:
"ugughuhruh" haha 😆 
The _tmp_ is to make sure, we only affecting `GroupEdit.vue` as this is a proposal. If we decide to change all forms (in a later PR), the _tmp_ things can be removed/cleaned-up.

Reply:
This will be cleaned-up in a later PR, which updates forms globally